hw/mcu/dialog/da1469x/include/mcu/da1469x_hal.h

```diff
@@ -68,6 +68,22 @@ struct da1469x_hal_spi_cfg {
     int8_t pin_ss;
 };
 
+/**
+ * Allow QSPI and the connected flash to be setup.
+ * This allows proper operation when configation script is not
+ * executed/present yet.
+ *
+ * @param dev    Pointer to device
+ * @param bursta BURSTCMDA_REG register setting
+ * @param burstb BURSTCMDB_REG register setting
+ * @param cmds   Array of byte sized commands sent over SPI to flash
```

Comment:
technically speaking looking in the the code only one command is sent (with multiply bytes).